packages/harmony/src/components/nav-item/NavItem.tsx

```diff
@@ -92,6 +92,7 @@ export const NavItem = ({
             lineHeight='single'
             color={textColor}
             ellipses
+            css={{ flex: 1 }}
```

Comment:
what's up with this?
also on line 82, is `flex: 1` not set by default on `Flex`?

Reply:
the items in the playlist weren't responding properly to justify-content